### ISSUE TITLE
Future of Questie?

### ISSUE THREAD

**Issue by Gogo1951** [Questie/Questie#3423]
Hey @AeroScripts & @BreakBB ,

Wrath will be here before we know it, what's in store for Questie?

Questie has gone all of TBC without any feature changes, only minor bug fixes and data updates. 

My impression is that @AeroScripts is 100% done with WoW, and @BreakBB wants to retain "control" but doesn't have any interest in doing the work to add new features. Am I wrong?

Questie is a really important add-on and rather than let it stagnate or rot, it seems like it would be good to get some fresh blood in.

You guys have done great getting it to what it is, but if you don't have interest or time to work on this any more ... what are you doing to find your replacements?

**Comment by BreakBB** [comment]
Hey Gogo, long time no see 👋🏻 

You are right, Questie didn't receive any "absolutely major features" and none are already planned out for the Wrath release tbh. The main problem (as you know) still is the limit in time and people. I've been the only active developer for the past year or so, but I am very grateful for the awesome help of @Laumesis and the great PRs we received throughout the expansion from many different people!

As so the Questie team always appreciate any help it gets and we are open for people willing to invest their time developing a free addon like Questie. The team and me don't see the need for "replacements" of our person, but it would be fantastic if new people would join the team so Questie will become even better 🎉

But hence it is a hobby for most people (just like me) it not always has the highest priority and we can understand most WoW players are not willing to invest a great amount of time developing an addon besides their gaming and personal life.

Once the Wrath beta hits we will see who crawls out of their holes and decide then what is possible and what not 😁 Till then we will continue as before and ensure Questie is in good shape and as helpful as possible 👍🏻

**Comment by Muehe** [comment]
Hey Gogo, hope you are doing good. While you are here, we have been contacting Patreon support because of the account transfer. They said they could do it for you and will be contacting you. Not sure if aero forwarded my message on Discord (couldn't DM you), would appreciate if you could check whatever e-mail is associated with that account. It's been a year now, would really like to get those empty promises removed.

**Comment by Gogo1951** [comment]
@BreakBB - If you don't have time to do development on this, why stay involved? It seems like people on Discord don't know that Questie is basically abandoned... just sort of limping along without anyone at the helm who wants to work on it. Probably good to put the word out that you're all looking for someone new who can take over.

@Muehe - @AeroScripts has always had the owner account on Patreon. I don't think I have access any more even. I gave him the email, password, and 2FA backup codes. And when Patreon asked, I told them they could release that to Luke. 

If you guys are just camping this out of... vanity? Maybe keep a contributed credit and move on to something you find more interesting.

Just feels like you're intent on camping this until it dies... really sad way for a great add-on to go. Like anything, software needs people to pluck the weeds and plant new things. If that's not you, then it's good to just make room for the next guy.

**Comment by Gogo1951** [comment]
And @Muehe, as for "empty promises" -- there is code in one of Luke's branches that do all the things talked about on Patreon. It was just a matter of you guys not wanting to do any more work than was absolutely necessary to keep the lights on.

**Comment by Muehe** [comment]
I don't want to rip open old wounds here, but it has been sitting there like that for over a year now with nobody bothering to do anything about it. That's an empty promise.

As for "camping this", I'm not really sure what you mean. What big features are we missing? Have we shot down any PRs I don't know about?

**Comment by Gogo1951** [comment]
Yes, you have shot down PRs because you guys are hell-bent on not taking on any more work than is absolutely necessary to keep the lights on. https://github.com/Questie/Questie/pull/2686

You have to ask yourself, "Is this really what's good for the project?" And if not... maybe thinks about just retiring.

That's what I mean by camping this. You're sitting on it, not lifting a finger to improve it, and not letting others improve it either. It's a shame.

**Comment by BreakBB** [comment]
There is plenty of room for new people, but we are also not explicitly looking for people taking over the project. Whoever is interested in working on Questie can reach out to us.

We made quite some mistakes about trusting people joining the team without them prooving themself towards us. We won't make that mistake again, but again: We are very open for new people 😁

Also Questie is far from abandoned, even though it is not as actively developed as other addons maybe. If people want to change anything about it: That is what PRs are for 🤷🏻

To be straigth to you: I am tired of discussing with you, because it leads to nothing good. Not for you and even more not for us. If you want to improve Questie get yourself opening up PRs and stop being mean to others trying to keep a great project alive.

**Comment by Muehe** [comment]
Just for reference, I knew about #2686

If you think not merging it is unfair then you should still be able to discuss that on the PR or open an issue about that specifically.
